rotkehlchen/chain/manager.py

```diff
@@ -1527,3 +1527,18 @@ def get_loopring_balances(self) -> Dict[Asset, Balance]:
                 aggregated_balances[asset] += balance
 
         return dict(aggregated_balances)
+
+    def get_chain_manager(
+        self,
+        blockchain: SupportedBlockchain,
```

Comment:
Indentation level + 1 here.

It's unfortunately not enforced by the linter.

Also specify the type more accurately.
```suggestion
            self,
            blockchain: Literal[SupportedBlockchain.ETHEREUM, SupportedBlockchain.KUSAMA, SupportedBlockchain.POLKADOT, SupportedBlockchain.AVALANCHE],
```

Reply:
:+1: done